ginisdk/src/androidTest/java/net/gini/android/SdkIntegrationTestAccounting.java

```diff
@@ -0,0 +1,361 @@
+package net.gini.android;
+
+import static net.gini.android.helpers.TrustKitHelper.resetTrustKit;
+
+import android.content.Context;
+import android.content.res.AssetManager;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.Build;
+import android.support.test.filters.LargeTest;
+import android.support.test.filters.SdkSuppress;
+import android.test.AndroidTestCase;
+import android.util.Log;
+
+import com.android.volley.toolbox.NoCache;
+
+import net.gini.android.DocumentTaskManager.DocumentUploadBuilder;
+import net.gini.android.authorization.EncryptedCredentialsStore;
+import net.gini.android.authorization.UserCredentials;
+import net.gini.android.helpers.TestUtils;
+import net.gini.android.models.Document;
+import net.gini.android.models.SpecificExtraction;
+
+import org.json.JSONException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import bolts.Continuation;
+import bolts.Task;
+
+@LargeTest
+public class SdkIntegrationTestAccounting extends AndroidTestCase {
+
+    private Gini gini;
+    private String clientId;
+    private String clientSecret;
+    private String apiUriAccounting;
+    private String userCenterUri;
+
+    @Override
+    protected void setUp() throws Exception {
+        final AssetManager assetManager = getContext().getResources().getAssets();
+        final InputStream testPropertiesInput = assetManager.open("test.properties");
+        assertNotNull("test.properties not found", testPropertiesInput);
```

Comment:
Is this `assert` needed during the set-up? I mean, having the input is a prerequisite and should not fail.

Reply:
It serves as a safety net and an early warning in case the `test.properties` file was not created. Gives a direct indication of the problem.